components/src/core/score-card/scorecard.component.scss

```diff
@@ -0,0 +1,77 @@
+@import '~@pxblue/angular-themes/theme.scss';
+@import '~@pxblue/colors/palette.scss';
+
+.root {
+    padding: 0;
+}
+
+// Styles applied to supplied action items.
+:host ::ng-deep .action-items * {
+    margin-left: 12px;
+    cursor: pointer;
+}
+
+.header {
+    border-top-left-radius: 4px;
```

Comment:
is this a roundness that exists in the theme?

Reply:
I've updated this to inherit the border roundness from the mat card container.